src/services/sortingSchemes.js

```diff
@@ -1,60 +1,67 @@
+import React, { Fragment } from "react"
+import { Trans, SelectOrdinal } from '@lingui/macro';
+import * as common from './commonTranslations';
+import { BsArrowUp, BsArrowDown } from 'react-icons/bs';
+
+const comboTricksTitle = (trickCount) => <SelectOrdinal id="sortingCategory.comboTricksTitle" value={trickCount} other="# Trick Combo" />;
+
 export const trickSortingSchemes = [
-  {"name": "Level Upwards",
+  {"name": <Fragment><Trans id={common.level.id} /> <BsArrowUp className="icon-white" /></Fragment>,
```

Comment:
I like the idea with the arrows!
But on the phone view the background is white and you can't really see the arrows.
We probably need to adapt the className depending on the screen mode.

Reply:
Nice I've just pushed a fix for that :)